### PR TITLE
Add spider for Livique, a Swiss furniture chain

### PR DESCRIPTION
**Pull request by brawer**

### CODE REVIEW

**Comment by iandees** [comment]
At least some of these locations have lat/lons. Can you try to extract those please?

**Comment by brawer** [comment]
> At least some of these locations have lat/lons. Can you try to extract those please?

Done.

File: locations/spiders/livique_ch.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+
+class LiviqueCHSpider(SitemapSpider, StructuredDataSpider):
+    name = "livique_ch"
+    brands = {
+        "Livique": "Q15851221",
+        "Lumimart": "Q111722218",
+    }
+    allowed_domains = ["www.livique.ch"]
+    sitemap_urls = ["https://www.livique.ch/sitemap.xml"]
```
Comment (Cj-Malone) [inline locations/spiders/livique_ch.py:15]:
```suggestion
    sitemap_urls = ["https://www.livique.ch/sitemap.xml"]
    sitemap_follow = ["/STORE-"]
```
We can avoid pulling the product/category sitemaps with the above.

**Review by Cj-Malone — APPROVED** [review]
LGTM, thank you.

**Pull request merged**